.github/workflows/check-sdk-examples.yaml

```diff
@@ -49,3 +44,22 @@ jobs:
           max_attempts: 5
           timeout_minutes: 20
           command: cd ./ecosystem/typescript/sdk/examples/javascript && pnpm install && pnpm test
+
+  run-python-examples:
+    runs-on: ubuntu-latest
+    env:
+      APTOS_NODE_URL: https://fullnode.devnet.aptoslabs.com
```

Comment:
I didn't realize the Python SDK was smart enough to add the `/v1`?

Reply:
I think you added that...

Reply:
lol okay go me